programs/programs/urgent_needs/urgent_need_functions.py

```diff
@@ -158,6 +161,45 @@ def eligible(cls, screen: Screen):
         Return True if the household is has an income bellow 200% FPL or someone in the household is over 60 years old
         '''
         fpl = FederalPoveryLimit.objects.get(year='THIS YEAR').as_dict()
-        is_income_eligible = screen.calc_gross_income('yearly', ['all']) < fpl[screen.household_size]
+        is_income_eligible = (
+            screen.calc_gross_income('yearly', ['all']) < fpl[screen.household_size]
+        )
         is_age_eligible = screen.num_adults(age_max=60)
         return is_income_eligible or is_age_eligible
+
+
+class CoEmergencyMortgageIncomeLimitCache(Cache):
+    expire_time = 60 * 60 * 24
+    default = {}
+
+    def update(self):
+        spreadsheet_id = '1M_BQxs135UV4uO-CUpHtt9Xy89l1RmSufdP9c3nEh-M'
+        range_name = "'100% AMI 2023'!A2:I65"
+        sheet_values = sheets_get_data(spreadsheet_id, range_name)
+
+        if not sheet_values:
+            raise Exception('Sheet unavailable')
+
+        data = {d[0] + ' County': [int(v.replace(',', '')) for v in d[1:]] for d in sheet_values}
+
+        return data
+
+
+class CoEmergencyMortgageAssistance(UrgentNeedFunction):
+    limits_cache = CoEmergencyMortgageIncomeLimitCache()
+    dependencies = ['income_amount', 'income_frequency', 'household_size', 'county']
+
+    @classmethod
+    def eligible(cls, screen: Screen):
+        income = int(screen.calc_gross_income('yearly', ['all']))
+
+        limits = CoEmergencyMortgageAssistance.limits_cache.fetch()
+
+        if screen.county not in limits:
+            return False
+
+        income_limit = limits[screen.county][
+            screen.household_size - 1
+        ]
+
+        return income < income_limit
```

Comment:
Do we want `<=`?

Reply:
@kbottagaro is going to go through the programs and check for this at some point. This is a big edge case, so I am going to merge as is.